dask/dataframe/methods.py

```diff
@@ -324,8 +325,24 @@ def size(x):
     return x.size
 
 
-def values(df):
-    return df.values
+def to_numpy(df, _dtype, na_value):
```

Comment:
I know this is just following the Pandas API, but this is problematic for cuDF and possibly any other future library that could live underneath Dask DataFrame.